website/sidebars.json

```diff
@@ -1,15 +1,15 @@
 {
   "docs": {
     "About": [
-      "about/introduction",
-      "about/getting-started"
+      "about/introduction"
     ],
-    "Installation": [
-      "installation/building",
-      "installation/running"
-
+    "Running": [
+      "installation/running",
+      "installation/configuration-options",
+      "installation/kubernetes"
     ],
     "Contributing": [
+      "contributing/building",
```

Comment:
This md doesn't exist.
Don't you want to keep the [section](https://github.com/ExpediaGroup/pitchfork/blob/master/docs/installation/building.md) on how to build?

Reply:
i'll group them together